### ISSUE TITLE
Error in example of dcat:mediaType

### ISSUE THREAD

**Issue by makxdekkers** [w3c/dxwg#170]
The way `dcat:mediaType` is used in the examples in section 4 of the existing Recommendation and in section 5 of the Editor's Draft is in error. The examples have:

`dcat:mediaType "text/csv" `

According to the definition of `dcat:mediaType`, its range is `dct:MediaTypeOrExtent`, not `rdfs:Literal`.

This should be corrected in the FPWD.

**Comment by akuckartz** [comment]
This also illustrates how confusing that property is :-/

**Comment by makxdekkers** [comment]
@akuckartz I personally don't think the property is confusing; the definition is quite clear. The main problem, it seems to me, is that implementers do not always read definitions and think they understand what a property is by just looking at the URI or label.

**Comment by dr-shorthair** [comment]
@makxdekkers could you provide example of how dcat:mediaType should be used? Should the value be a URI?

**Comment by jakubklimek** [comment]
@dr-shorthair @makxdekkers Isn't the usage illustrated in my examples in https://github.com/w3c/dxwg/issues/259, i.e. used in the Czech catalog and therefore in EDP, correct?
E.g.:
```turtle
<distribution> dcat:mediaType <http://www.iana.org/assignments/media-types/text/csv> .
```

**Comment by dr-shorthair** [comment]
Proposed fix in #303

**Comment by makxdekkers** [comment]
@jakubklimek Correct.

**Comment by dr-shorthair** [comment]
Fixed in #307
